site/ppan.cylc

```diff
@@ -18,7 +18,10 @@
             COPY_TOOL=gcp
 
     [[STAGE-HISTORY]]
-        pre-script = module load gcp/2.3 hsm/1.3.0
+        pre-script = """
+            module load gcp/2.3 hsm/1.3.0
+            module load fre/2025.02
```

Comment:
To save us from updating this each release, we're setting a FRE_VERSION jinja variable here:

https://github.com/NOAA-GFDL/fre-workflows/blob/main/flow.cylc#L4

So let's change that one there to 2025.02 and then use it here:

module load fre/{{ FRE_VERSION }}